python/mujincontrollerclient/realtimerobotclient.py

```diff
@@ -215,14 +215,16 @@ def GetGrabbed(self, timeout=10, **kwargs):
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def GetTransform(self, targetname, unit='mm', timeout=10, **kwargs):
+    def GetTransform(self, targetname, targetpath='', unit='mm', timeout=10, **kwargs):
         """gets the transform of an object
         :param targetname: name of the object
+        :param targetpath: string path of object separated by /, e.g. GP7/calibration_board, see realtimerobottask3.py::GetTransform(...); this parameter takes precedence over targetname
         :param unit: unit of the result translation
         :return: transform of the object in a json dictionary, e.g. {'translation': [100,200,300], 'rotationmat': [[1,0,0],[0,1,0],[0,0,1]], 'quaternion': [1,0,0,0]}
         """
         taskparameters = {'command': 'GetTransform',
                           'targetname': targetname,
+                          'targetpath': targetpath,
```

Comment:
use `linkname`, `geometryname` instead

Reply:
I actually have some additional parameters, so this is why I kept it as a `/` separated path string